source/routing/raptor.cpp

```diff
@@ -212,6 +212,11 @@ struct Dom {
     bool clockwise;
     typedef std::pair<size_t, StartingPointSndPhase> Arg;
     inline bool operator()(const Arg& lhs, const Arg& rhs) const {
+        if (lhs.second.count == rhs.second.count
```

Comment:
A comment might be nice to tell why we do that (you can get inspiration in the comment of that PR)